docs/modules/operation/pages/database-reports/database.adoc

```diff
@@ -28,6 +28,92 @@ The {page-component-title} Report Engine passes a JDBC connection to the {page-c
 
+[[ga-database-report-use-existing-report]]
+
+== Using existing reports
+
+OpenNMS provides the following sample report templates, located in `$\{OPENNMS_HOME}/etc/report-templates`, which you can customize to suit your needs:
+
+
+[options="header, autowidth"]
+|===
+| Report Name            | Category     | Description
+
+|  xref:operation:database-reports/templates/avail-summary.adoc[Availability Summary]
+| Status
+| One page summary showing summary statistics on outages and availability for user-specified surveillance category during a user-specified number of days.
+
+
+| xref:operation:database-reports/templates/availability.adoc[Availability by Node]
+| Status
+| List of availability statistics for each node in user-specified category during a user-specified number of days.
+
+| Default Calendar
+| Unknown
+| xxxx
+// QUESTION: Need help getting info on this report.  It is not a JasperReport.
+
+| Default Classic
+| Unknown
+| xxxx
+// QUESTION: Need help getting info on this report.  It is not a JasperReport.
+
+| xref:operation:database-reports/templates/event.adoc[Event Analysis]
+| Status
+| Tables and charts showing different slices on the top 25 events over the past 30 days.
+
+| xref:operation:database-reports/templates/grafana.adoc[Grafana Dashboards]
+| Grafana
+| Create Report from Grafana Dashboard, 3 options are provided.
+// ToDo: Get more detail on reports, find out why helper file with zoneid is not work; check to see if zoneid is depracated.
+
+| xref:operation:database-reports/templates/interface-avail.adoc[Interface Availability]
+| Status
+| Shows the availability of interfaces with outages within a time range.
+
+| xref:operation:database-reports/templates/maintenance-expired.adoc[Maintenance Contracts Expired]
+| Asset Management
+| Shows which assets with maintenance contracts are going to expire within a given time range.
+// COMMENT: The name of this report is misleading to me.
+
+| xref:operation:database-reports/templates/maintenance-stragegy.adoc#maintenance-strategy-report[Maintenance Contracts Strategy]
+| Asset Management
+| Provides an overview of assets, showing how many within each age group have an active contract, no contract or an expired contract; and shows how many will be expiring in the next 12 months.
+
+| xref:operation:database-reports/templates/morning.adoc[Early Morning Report]
+| Status
+| Summary of status of network nodes over the past 24 hours.
+
+| xref:operation:database-reports/templates/response-summary.adoc[Response Time Summary]
+| Status
+| Summary of response times across surveillance category.
+
+| xref:operation:database-reports/templates/response-time.adoc[Response Time by Node]
+| Status
+| Response times by node.
+
+| xref:operation:database-reports/templates/serial.adoc[Serial Interface Utilization Summary]
+| Status
+| Summary of serial interface utilization.
```

Comment:
I think utilization is a term of art and it tends to refer to a measurement of capacity.  Even though they are clearly synonyms, it sounds weird to me to switch it.  It could be just me.  The example that comes to mind is if someone asks about your "storage utilization" that would prompt a percentage or a capacity description, but "storage use" is ambiguous as it could mean, I use the storage for storing Word files and spreadsheets but no videos or data files.